lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -267,6 +267,42 @@ def request_params_hosts(params)
       end
     end
 
+    class ExportDefaultCommand < HammerCLIForeman::Command
+      include HammerCLIKatello::LocalHelper
+      include HammerCLIKatello::ApipieHelper
+
+      PUBLISHED_REPOS_DIR = "/var/lib/pulp/published/yum/https/repos/".freeze
+
+      desc _('Export the library default content view')
+
+      command_name "export-default"
+
+      success_message _("Default content view export is available at")
+      failure_message _("Could not export the default content view at")
+
+      option '--export-dir', 'EXPORT_DIR', _("Directory to put content view version export into.")
+
+      validate_options do
+        option(:option_export_dir).required
+      end
+
+      build_options
+
+      def execute
+        export_dir = options['option_export_dir']
+
+        Dir.mkdir(export_dir) unless Dir.exist?(export_dir)
+        result = Kernel.system("rsync -aL #{PUBLISHED_REPOS_DIR} #{export_dir}")
+        if result == true
+          puts "#{success_message} #{export_dir}"
+          return HammerCLI::EX_OK
+        else
+          puts "#{failure_message} #{export_dir}"
```

Comment:
You should be able to use `output.print_error` and `output.print_message`, defined [here](https://github.com/theforeman/hammer-cli/blob/master/lib/hammer_cli/abstract.rb#L163) and [here](https://github.com/theforeman/hammer-cli/blob/master/lib/hammer_cli/output/output.rb#L8).

Be sure to wrap your strings in `_("my string here")` for translations